data/sql_updates/committee_history.sql

```diff
@@ -78,9 +78,9 @@ select distinct on (fec_yr.cmte_id, fec_yr.fec_election_yr)
     f1.cand_pty_tp as party_type,
     f1.cand_pty_tp_desc as party_type_full,
     f1.qual_dt as qualifying_date,
-    dates.first_file_date,
-    dates.last_file_date,
-    dates.last_f1_date,
+    cast(cast(dates.first_file_date as text) as date) as first_file_date,
+    cast(cast(dates.last_file_date as text) as date) as last_file_date,
+    cast(cast(dates.last_f1_date as text) as date) as last_f1_date,
```

Comment:
The columns as in are numeric, and I seem to remember having errors casting numeric straight to a date.  I can try casting directly in my console to see if it will work.